templates/dev/mock-acs.html.twig

```diff
@@ -3,7 +3,9 @@
     <b>Note: </b> javascript is disabled, please click the button below to proceed
 </noscript>
 One moment please...
-
+<br/>
+TODO: Add emailaddress field to use instead of NameID
```

Comment:
Is this work in progress or work for some other time? Maybe open a ticket if this should be addressed at a later point.

Reply:
Took some effort to do it, now so it would become better testable and I also added a couple of functional tests.